src/components/static/Navigation.svelte

```diff
@@ -122,16 +122,16 @@
   </div>
     {#if !isMenuHidden}
       <!-- STANDARD NAVBAR -->
-      <div class="hidden mx-24 my-4 text-white lg:block xl:block 2xl:block h-96">
-        <div class="flex justify-center">
+      <div class="hidden text-white lg:block xl:block 2xl:block h-96" style="margin-left: 12.5%; margin-right: 12.5%;">
+        <div class="flex">
           {#each dropdownLinks as dropdown}
           <div
-            class="mt-4 transition transform hover:text-mcswf-gold hover:scale-110 px-14"
+            class="mt-4 transition transform hover:text-mcswf-gold hover:scale-110 pr-10"
             on:mouseleave={() => closeDropdown(dropdown.text)}
             on:mouseenter={() => openDropdown(dropdown.text)}
           >
             <a href={dropdown.url}>
-              <h2 class="font-bold">{dropdown.text}</h2>
+              <h2 style="font-size:20px" class="pb-1 font-bold">{dropdown.text}</h2>
```

Comment:
@whalenda used `text-sm`, it is equivalent to 20px.
https://tailwindcss.com/docs/font-size

Reply:
updated to text-xl which is 20px